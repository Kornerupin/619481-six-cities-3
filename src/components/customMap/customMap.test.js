import React from "react";
import renderer from "react-test-renderer";
import CustomMap from "./customMap";

const testData = [
  {
    id: 34,
    mark: `Premium`,
    coords: [52.369553943508, 4.85309666406198],
    img: {
      imgLink: `apartment-01.jpg`,
      imgAlt: `title 0`,
    },
    link: `#`,
    price: {
      priceValue: `â‚¬120`,
      priceText: `/ night`,
    },
    rating: 4,
    title: `Beautiful & luxurious apartment at great location`,
    type: `Apartment`,
  }
];

it(`<CustomMap /> component render test`, () => {
  const tree = renderer.
    create(<CustomMap
      offers={testData}
    />).parseJSON;

  expect(tree).toMatchSnapshot();
});
