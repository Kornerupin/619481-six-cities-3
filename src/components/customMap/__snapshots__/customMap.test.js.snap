// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomMap /> component tests Render test 1`] = `undefined`;
